GitHub Action fails with exit 127: `.venv/bin/prowler: no such file` (workdir override breaks relative ENTRYPOINT)

## Summary

The composite action in `action.yml` sets `-w /home/prowler/workspace` on its `docker run`. That overrides the image's default workdir (`/home/prowler`) where the relative ENTRYPOINT `.venv/bin/prowler` correctly resolves. With the override, docker looks for `/home/prowler/workspace/.venv/bin/prowler` — which doesn't exist; the venv lives at `/home/prowler/.venv/`. The container exits 127 before any provider code runs.

This affects any caller of the action that triggers a real scan (i.e. anyone who isn't only smoke-testing the Validate-inputs step).

## Reproduction

```yaml
- uses: prowler-cloud/prowler@v5.27.0
  with:
    provider: github
    image-tag: "5.27.0"
    flags: --organization <your-org>
  env:
    GITHUB_PERSONAL_ACCESS_TOKEN: ${{ secrets.YOUR_TOKEN }}
```

Failing run output:

```
docker: Error response from daemon: failed to create task for container:
  failed to create shim task: OCI runtime create failed: runc create failed:
  unable to start container process: error during container init:
  exec: ".venv/bin/prowler": stat .venv/bin/prowler: no such file or directory
##[error]Process completed with exit code 127.
```

Reproduces identically when invoking docker manually with the same arguments the action constructs:

```bash
docker run --rm \
  -e GITHUB_PERSONAL_ACCESS_TOKEN=ghs_… \
  -v "$PWD:/home/prowler/workspace" \
  -v "$PWD/output:/home/prowler/workspace/output" \
  -w /home/prowler/workspace \
  prowlercloud/prowler:5.27.0 \
    github --organization YOUR_ORG --output-formats json-ocsf
```

The same docker invocation **without** `-w /home/prowler/workspace` succeeds.

## Root cause

`docker inspect prowlercloud/prowler:5.27.0`:

```json
{
  "Entrypoint": [".venv/bin/prowler"],
  "WorkingDir": "/home/prowler"
}
```

`ls -la /home/prowler/` inside the container shows `.venv/` at `/home/prowler/.venv/` — not at `/home/prowler/workspace/.venv/` (which doesn't exist; it's just the mount target).

Setting workdir to `/home/prowler/workspace` makes the JSON-array ENTRYPOINT `.venv/bin/prowler` resolve relative to that workdir → `/home/prowler/workspace/.venv/bin/prowler` → not found → exit 127.

The action's [`action.yml`](https://github.com/prowler-cloud/prowler/blob/master/action.yml) lines (at v5.27.0 SHA `baaf56e`):

```yaml
docker run --rm \
  "${env_args[@]}" \
  -v "$GITHUB_WORKSPACE:/home/prowler/workspace" \
  -v "$GITHUB_WORKSPACE/output:/home/prowler/workspace/output" \
  -w /home/prowler/workspace \    # <-- this line breaks the entrypoint
  "prowlercloud/prowler:${INPUT_IMAGE_TAG}" \
  "$INPUT_PROVIDER" \
  ...
```

## Suggested fix

Either (a) drop the `-w` flag so docker uses the image's default workdir `/home/prowler`, or (b) make the ENTRYPOINT absolute in the Dockerfile (`/home/prowler/.venv/bin/prowler`).

Option (a) is the smaller change and keeps the image self-describing. Output paths in flags that users pass would need to be tolerant of the new workdir, but the image already mounts `/home/prowler/output` separately, so output discovery via `find output/` in subsequent steps still works.

## Workaround

Bypass the composite action and invoke docker directly:

```yaml
- name: Run Prowler GitHub scan
  env:
    GITHUB_PERSONAL_ACCESS_TOKEN: ${{ steps.app-token.outputs.token }}
  run: |
    mkdir -p output
    docker run --rm \
      -e GITHUB_PERSONAL_ACCESS_TOKEN \
      -v "$PWD/output:/home/prowler/output" \
      -v "$PWD/.github/prowler/mutelist.yaml:/home/prowler/mutelist.yaml:ro" \
      prowlercloud/prowler@sha256:8589f9cc83a0c1ed1d0f443ce3c11b156507909ef0d6d06a45739cc634ab2605 \
        github --organization <org> --output-formats csv json-ocsf html \
        --mutelist-file /home/prowler/mutelist.yaml
```

This works because the image's default workdir `/home/prowler` is preserved.

## Environment

- Action ref: `prowler-cloud/prowler@v5.27.0` (commit `baaf56ea5ef13e3861fe27aee36867f81579fad9`)
- Image: `prowlercloud/prowler:5.27.0` (`sha256:8589f9cc83a0c1ed1d0f443ce3c11b156507909ef0d6d06a45739cc634ab2605`)
- Runner: `ubuntu-24.04` and macOS Docker Desktop locally — both reproduce.
- Provider: `github` (also testable with other providers; the bug is in container init, before any provider logic).

Happy to send a PR if there's interest in option (a).

--- a/Dockerfile
+++ b/Dockerfile
@@ -100,4 +100,4 @@ RUN pip uninstall dash-html-components -y && \
     pip uninstall dash-core-components -y
 
 USER prowler
-ENTRYPOINT [".venv/bin/prowler"]
+ENTRYPOINT ["/home/prowler/.venv/bin/prowler"]

--- a/prowler/CHANGELOG.md
+++ b/prowler/CHANGELOG.md
@@ -17,6 +17,10 @@ All notable changes to the **Prowler SDK** are documented in this file.
 
 - `OktaProvider.test_connection` accepts an optional `provider_id` (org domain) and raises `OktaInvalidProviderIdError` (14007) when it doesn't match the authenticated org — guards against stored UID drifting from the credentials' org [(#11184)](https://github.com/prowler-cloud/prowler/pull/11184)
 
+### 🐞 Fixed
+
+- Container image now uses an absolute `ENTRYPOINT` (`/home/prowler/.venv/bin/prowler`) so it works under any runtime `--workdir`. The relative entrypoint was breaking the official GitHub Action (`prowler-cloud/prowler@v5.27.0`) and any `docker run` with a custom `-w` [(#XXXXX)](https://github.com/prowler-cloud/prowler/pull/XXXXX)
+
 ---
 
 ## [5.27.1] (Prowler v5.27.1)

--- a/prowler/CHANGELOG.md
+++ b/prowler/CHANGELOG.md
@@ -19,7 +19,7 @@ All notable changes to the **Prowler SDK** are documented in this file.
 
 ### 🐞 Fixed
 
-- Container image now uses an absolute `ENTRYPOINT` (`/home/prowler/.venv/bin/prowler`) so it works under any runtime `--workdir`. The relative entrypoint was breaking the official GitHub Action (`prowler-cloud/prowler@v5.27.0`) and any `docker run` with a custom `-w` [(#XXXXX)](https://github.com/prowler-cloud/prowler/pull/XXXXX)
+- Container image now uses an absolute `ENTRYPOINT` (`/home/prowler/.venv/bin/prowler`) so it works under any runtime `--workdir`. The relative entrypoint was breaking the official GitHub Action (`prowler-cloud/prowler@v5.27.0`) and any `docker run` with a custom `-w` [(#11313)](https://github.com/prowler-cloud/prowler/pull/11313)
 
 ---
 

